helpers/compile/configs.ts

```diff
@@ -18,7 +18,7 @@ export const adapterConfig: BuildOptions[] = [
     entryPoints: ['src/index.ts'],
     outfile: 'dist/index',
     outExtension: { '.js': '.mjs' },
-    emitTypes: false,
```

Comment:
Are the types different or is it just about the extension of the file (in which case they could be duplicated instead of re-bundled)?

Reply:
I think right now they are identical. They have to be separate files though, pointing both of them to the same `.d.ts` file will not work according to TS docs.

Reply:
Sure, created a follow up issue https://github.com/prisma/team-orm/issues/589